src/js/components/Menu/Menu.js

```diff
@@ -32,10 +32,14 @@ class Menu extends Component {
     justifyContent: 'start',
   };
 
-  state = { activeItemIndex: -1, open: false };
-
   buttonRefs = {};
 
+  constructor(props) {
+    super(props);
+
+    this.state = { activeItemIndex: -1, open: props.open || false };
```

Comment:
should we add on the description defaults to `false`?

Reply:
yeap!